src/static/js/accounts_trx.js

```diff
@@ -279,6 +280,15 @@ $(function() {
                 const row = common.get_row(controller.rows, $(this).attr("data-id"));
                 validate.reset("dialog-edit");
                 $("#trxid").val(row.ID);
+                $("#supplier").personchooser("clear");
+                if (row.ANIMALCOSTID || row.OWNERDONATIONID) {
+                    $("#supplierrow").hide();
+                } else {
+                    $("#supplierrow").show();
+                }
+                if (row.OWNERID) {
+                    $("#supplier").personchooser("loadbyid", row.OWNERID);
+                }
```

Comment:
The link to this person needs displaying in the render_tablebody function so that the user can see it in the list of transactions